### DISCUSSION TITLE
[Feature] Select all albuns on album selection

### DISCUSSION THREAD

**Discussion by JoseFilipeFerreira** [immich-app/immich#12739] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Having the option of selecting all albums for backup on the selection screen would streamline adding new albums for users with too many albums such as myself.
This would ignore adding albums previous selected as "do not import".

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by lesett** [comment]
Yeah this would be especially awesome for sharing all your albums to family members more efficently!

**Comment by davidSGE** [comment]
This definetly has to be added. I got 602 albums in my iCloud library...
Also there should be:
- a checkmark to automatically include new albums created in future
- an sorting button to sort the albums alphabetically. Right now it's impossible to find the album I'm looking for. In other apps it is similar, I guess iOS is returning the albums in a weird order. But thats why there has to be a sorting button

**Comment by maslanypotwor** [comment]
Bumping this as I would love to have this feature. Currently selecting all albums on my mobile and when adding new user (and they have like 150 albums in their iCloud) is a huuuuge pain. And it could be solved by a single "select all" button. 

What would be awesome is if it could always take also newly created albums after the "select all" button was clicked and ignore one only after user explicitly selected to not backup particular one.

**Comment by masx200** [comment]
https://github.com/immich-app/immich/discussions/15636

**Comment by Ben-Diehlci** [comment]
Bumping this, this should definitely be a feature. The ability to just have a checkbox that says “Backup All Future and Current Photo and Videos”.

**Comment by NicholasFlamy** [comment]
While the Recents album exists, currently it's useless if you want to sync your albums. Also, having this option includes all future albums is also needed.

**Reply by NicholasFlamy** [reply]
Related to #18607

**Comment by RealFascinated** [comment]
Would love to see this added

**Comment by mlivingood** [comment]
+1 on all of this. Especially the option to automatically sync new/future albums. New users on my server won't remember to go and "select" any new albums or shared albums they have from their phone for backup, so having it automatically include them would be awesome.

**Comment by NicholasFlamy** [comment]
The feature mentioned in the original post has been implemented in the timeline beta in v1.136.0 and up.

There is a lot of interest in having this include newly created albums as well so I created #20568.

**Comment by NicholasFlamy** [comment]
Apparently this is still not available on iOS according to this comment: https://github.com/immich-app/immich/discussions/20568#discussioncomment-14133790

**Comment by danb35** [comment]
Another vote for this (on iOS, specifically).  What's the point of the "exclude album" setting without it?

If you can "select all" or "back up all albums," then being able to exclude specific albums make sense; it's then "back up all albums except those specifically excluded."  But if you have to individually select each album to back up/sync, the "exclude" marker seems entirely superfluous.

**Comment by ViiFii** [comment]
+1 vote for this option. Moved all photos from nas storage to immich and now I've got ~700 albums. Need share that family photos within family as every albun name is YYY-MM-DD

**Reply by NicholasFlamy** [reply]
Are you on iOS? What version of the app?
